src/App.jsx

```diff
@@ -14,33 +17,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const RESET_SORTING = 0;
+const ALPHABETICAL_SORTING = 1;
+const SORT_BY_LENGTH = 2;
```

Comment:
The constants `RESET_SORTING`, `ALPHABETICAL_SORTING`, and `SORT_BY_LENGTH` are well-defined for managing sorting states. However, ensure that the `RESET_SORTING` state is handled explicitly in the `prepareGoods` function if it requires specific behavior beyond returning the original order.